example/HelloWorldFunction/src/main/java/helloworld/App.java

```diff
@@ -33,12 +38,20 @@
 
     @PowertoolsLogging(logEvent = true, samplingRate = 0.7)
     @PowertoolsTracing(captureError = false, captureResponse = false)
+    @PowertoolsMetrics(namespace = "EMF", captureColdStart = true)
```

Comment:
Can you change "EMF" to something which will help customers understand what a namespace is in this context. 

"Payments" or "Fraud & Risk"

I know it's a bit of a stretch when the function is called HelloWorld. Maybe that's something we can address in the future.